conans/test/functional/editable_reference/create_editable_package_test.py

```diff
@@ -0,0 +1,131 @@
+# coding=utf-8
```

Comment:
``functional/editable/mytest.py`` would be a nice "editable" package name for this category.